iOSDFULibrary/Classes/Implementation/SecureDFU/Services/SecureDFUService.swift

```diff
@@ -461,14 +461,14 @@ import CoreBluetooth
      
      - parameter report: method called when an error occurred
      */
-    func jumpToBootloaderMode(onError report: @escaping ErrorCallback) {
+    func jumpToBootloaderMode(dontSetAdvertisingName: Bool, onError report: @escaping ErrorCallback) {
```

Comment:
Same here, dontSetAdvertisingNAme: disableAlretnativeAdvertisingName is a bit confusing and unreadable in a way.

can it be renamed to :

jupToBootloaderMode(withAlternativeAdvertisingName: Bool, ...)

or something similar ?